sdk/tests/deqp/functional/gles3/es3fNegativeShaderApiTests.js

```diff
@@ -521,7 +521,7 @@ goog.scope(function() {
                 if (vec4_v == null)
                     assertMsgOptions(false, 'Failed to retrieve uniform location', false, true);
 
-                /** @type {Float32Array} */ var data = new Float32Array(8);
+                /** @type {Float32Array} */ var data = new Float32Array(12);
```

Comment:
If the length of the vector is 8. It is not an integer multiple of the assigned type for uniform3fv() below. Then an INVALID_VALUE may generated according to WebGL spec. 

if the vector length is 12, then it is an integer multiple of the assigned type for all APIs below.